README.md

```diff
@@ -26,7 +26,7 @@ See the **[XL Release Documentation](https://docs.xebialabs.com/xl-release/index
 
 # Overview #
 
-The xlr-xldeploy-plugin is a XL Release plugin that allows to start a control task, start a deployment, migrate a deployment package to another server or get the latest version of an application on XL Deploy. This plugin is now deprecated and [xlr-xld-plugin](https://docs.xebialabs.com/xl-release/how-to/xld-plugin.html) is the officially supported plugin.
+The xlr-xldeploy-plugin is a XL Release plugin that allows to start a control task, start a deployment, migrate a deployment package to another server or get the latest version of an application on XL Deploy. This plugin is now deprecated since XL Release 8.1 and up and [xlr-xld-plugin](https://docs.xebialabs.com/xl-release/how-to/xld-plugin.html) is the officially supported plugin.
```

Comment:
Will the official plugin only work in XLR 8.1?

Reply:
The official plugin supports all the community task starting 8.1 and up. It had lesser features in the earlier versions.